functional-samples/cookbook.push/background.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code is a modified version of
+// https://codelabs.developers.google.com/codelabs/push-notifications
+
+// These methods are meant to be called by the developer following the cookbook.
+/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "subscribeUser" }]*/
+
+const applicationServerPublicKey = '<key>';
+
+// Whether the extension will attempt to show a notification or not. Controlled
+// by subscribeUserVisibleOnlyFalse() and subscribeUserVisibleOnlyTrue(). Not
+// meant to be used in production due to ephemeral nature of service workers.
```

Comment:
"Not meant to be used in production due to ephemeral nature of service workers." - why is it in the sample?